submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/model/SubmitWebRequest.kt

```diff
@@ -32,6 +32,9 @@ data class SubmissionConfig(
     val storageMode: StorageMode?,
 )
 
+/**
+ * TODO: merge with ac.uk.ebi.biostd.client.integration.web.SubmissionFilesConfig as it is duplicated.
```

Comment:
Should we create a ticket for this or add it to https://www.pivotaltracker.com/story/show/183068218 ?